### DISCUSSION TITLE
Star exchange ⭐️⭐️⭐️⭐️⭐️

### DISCUSSION THREAD

**Discussion by bacca87** [Schweinepriester/github-profile-achievements#350] [General]
If you star my repo, i'll star yours back!

Daily check!

https://github.com/bacca87/great2

**Comment by cccy0** [comment]
https://github.com/cccy0/cccy0   thank u

**Reply by bacca87** [reply]
done!

**Reply by aziz0x48** [reply]
@cccy0 done https://github.com/c99tn/vMass

**Reply by WHAHA-HA** [reply]
@cccy0, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by sbsoso0411** [reply]
@cccy0, done.
https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by longquoc47krb** [reply]
You star me, I give it back. Thanks.
https://github.com/longquoc47krb/BookingEventTicket

**Reply by montasim** [reply]
Done @c99tn:
Do it for me, please.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
@cccy0 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Presstomsk** [reply]
Please, star me https://github.com/Presstomsk/AviaTickets

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by LPHBackspace** [reply]
hi

**Comment by hitblast** [comment]
@bacca87 Done.

https://github.com/IgKniteDev/IgKnite thanks in advance :D

**Reply by bacca87** [reply]
done!

**Reply by aziz0x48** [reply]
https://github.com/c99tn/vMass @hitblast done

**Reply by WHAHA-HA** [reply]
@hitblast, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by sbsoso0411** [reply]
@hitblast , done!
https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by hitblast** [reply]
@fullstack-webdev @WHAHA-HA Done as well :D

**Reply by ArielMAJ** [reply]
@hitblast I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by hitblast** [reply]
@ArielMAJ Done!

**Reply by ArielMAJ** [reply]
@hitblast ty!

**Comment by aziz0x48** [comment]
@bacca87 done https://github.com/c99tn/vMass

**Reply by bacca87** [reply]
done!

**Reply by WHAHA-HA** [reply]
Done, https://github.com/kingsman-fullstackdev/Audiophile

**Reply by sbsoso0411** [reply]
@c99tn , done!
https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by ArielMAJ** [reply]
@c99tn I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by WHAHA-HA** [comment]
@bacca87
Already done! https://github.com/kingsman-fullstackdev/Audiophile

**Reply by bacca87** [reply]
done!

**Reply by ArielMAJ** [reply]
@WHAHA-HA I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Ashu-Barnwal** [comment]
DONE 
https://github.com/Ashu-Barnwal/Color-Plate

**Reply by bacca87** [reply]
Done!

**Reply by sbsoso0411** [reply]
@Ashu-Barnwal, done!
https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by ArielMAJ** [reply]
@Ashu-Barnwal I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by bacca87** [comment]
up

**Comment by polekstulod** [comment]
Done. https://github.com/polekstulod/foodtrip-api

**Reply by bacca87** [reply]
done

**Reply by sbsoso0411** [reply]
@polekstulod, done!
https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by ArielMAJ** [reply]
@polekstulod I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by ghostinthematrix0** [comment]
https://github.com/mendeeislife/mendeeislife

**Reply by bacca87** [reply]
done

**Reply by bacca87** [reply]
I removed my star from your repo. You did not starred mine and this is unfair!

**Reply by aziz0x48** [reply]
same I removed my star too @mendeeislife

**Reply by sbsoso0411** [reply]
@mendeeislife , done!
https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by ArielMAJ** [reply]
@mendeeislife  I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by sbsoso0411** [comment]
https://github.com/fullstack-webdev/Noexam-WordPress
Thank you

**Reply by aziz0x48** [reply]
@fullstack-webdev done thanks

**Reply by bacca87** [reply]
@fullstack-webdev no, you did not.

**Reply by sbsoso0411** [reply]
@bacca87 , I've starred your repo. Please star my repo, thank you.

**Reply by bacca87** [reply]
@fullstack-webdev  done

**Reply by ArielMAJ** [reply]
@sbsoso0411  I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by 41pha1** [comment]
Done thanks in advance
https://github.com/41pha1/MIDI-Converter

**Reply by ArielMAJ** [reply]
@41pha1 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by montasim** [comment]
Done @bacca87:
Do it for me, please.
https://github.com/montasim/montasim

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by ArielMAJ** [comment]
@bacca87 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by ArielMAJ** [reply]
Already starred you @bensonarafat, waiting for you to star back!

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by bensonarafat** [comment]
https://github.com/codepymdev/grad

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Comment by Rantoryu** [comment]
@bacca87 
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!

**Reply by Koustav-Dey** [reply]
Done Please Check my profile

**Comment by Koustav-Dey** [comment]
Please star my repo: https://github.com/Koustav-Dey/MIni_Lang
I will star back anyone in the comment to this.
Thanx
